network-api/app/networkapi/templates/pages/menus/side-menu-items.html

```diff
@@ -4,9 +4,9 @@
     {% if page.is_current_or_ascendant or page.is_current_child or page.is_current_sibling and page.branch_level > 1 %}
       {% if page.branch_level > 0 %}
         {% if page.has_children or page.branch_level > 1 %}
-          <li>
```

Comment:
I'm not familiar of this codebase - just wondering why we are changing this from `li` to `div`